client/src/Utils/melody.js

```diff
@@ -1,27 +1,27 @@
 import Tone from "tone";
 
-export const NOTE_LOWEST = 12; // F3
-export const NOTE_HIGHEST = 27; // G5
+// TODO: enforce constraint in backend for this note range.
```

Comment:
Can we remove the TODO? I think the vocal range rule does this. Doesn't hurt doing a second layer check client side too though.

Reply:
There's no guarantee that the melodies will follow those rules because that requires a mutation to have happened; this is especially a problem with randomly generated melodies.
I'd like to have some constants for note range and maybe enforce it in the `Note`, but I think it'd be out of scope for this PR

Reply:
Ah right. damn does random have NO constraints???? We can't just make random create notes in the right interval (there's a numeric value that maps to each note now right?)

Reply:
It does (it uses [starting set](https://github.com/MichaelAEden/BotBand/blob/master/server/src/ga/GaWorker.ts#L146)), but it has different constraints than the rules. That's why I'd prefer to have one set of constants then 2 and enforce that rule in `Note` rather than GA

Reply:
makes sense to me 👍